RecoEgamma/EgammaElectronAlgos/interface/TrajSeedMatcher.h

```diff
@@ -290,8 +272,8 @@ class TrajSeedMatcher {
   std::unordered_map<int, TrajectoryStateOnSurface> trajStateFromVtxPosChargeCache_;
   std::unordered_map<int, TrajectoryStateOnSurface> trajStateFromVtxNegChargeCache_;
```

Comment:
change these too?

Reply:
These are different, as the key is just `int` and not `pair<int, GlobalPoint const&>`.